src/Controls/tests/TestCases.Shared.Tests/Tests/Issues/Issue20706.cs

```diff
@@ -0,0 +1,29 @@
+﻿#if TEST_FAILS_ON_CATALYST // Stepper interaction is not implemented on catalyst
+using NUnit.Framework;
+using UITest.Appium;
+using UITest.Core;
+
+namespace Microsoft.Maui.TestCases.Tests.Issues
+{
+	public class Issue20706 : _IssuesUITest
+	{
+		public override string Issue => "Stepper doesn't change increment value when being bound to a double in MVVM context (Windows)";
+		public Issue20706(TestDevice device) : base(device)
+		{
+		}
+
+		[Test]
+		[Category(UITestCategories.Stepper)]
+		
+		public void ChangeIncrementValue()
+		{
+			App.WaitForElement("incrementButton");
+			App.IncreaseStepper("myStepper");
+			App.Click("incrementButton");
+			App.IncreaseStepper("myStepper");
+			App.DecreaseStepper("myStepper");
+			VerifyScreenshot();
```

Comment:
Incrementing and decrementing hides the actual change. It may have stepped 10 or it may have stepped 2. Could the screenshot rather be directly after the increment.

```suggestion
			App.IncreaseStepper("myStepper");
			VerifyScreenshot();
```

Reply:
Actually, maybe don't bother with screenshots, just read the value from the entry. It will just make the test less flakey or reduce the number of images and weird things with entry cursors.

```cs
// compare text (should be 2)
// increment
// compare text (should be 12)
// decrement
// compare text (should be 2)

Reply:
I’ve updated the test to read the value directly from the Entry instead of relying on screenshots.